components/services/bat_ads/bat_ads_client_mojo_bridge.cc

```diff
@@ -336,6 +224,147 @@ void BatAdsClientMojoBridge::Log(
   bat_ads_client_->Log(file, line, verbose_level, message);
 }
 
+bool BatAdsClientMojoBridge::GetBooleanPref(
+    const std::string& path) const {
+  bool value = false;
+
+  if (!connected()) {
+    return value;
+  }
+
+  bat_ads_client_->GetBooleanPref(path, &value);
+  return value;
+}
+
+void BatAdsClientMojoBridge::SetBooleanPref(
+    const std::string& path,
+    const bool value) {
+  if (!connected()) {
+    return;
+  }
+
+  bat_ads_client_->SetBooleanPref(path, value);
+}
+
+int BatAdsClientMojoBridge::GetIntegerPref(
+    const std::string& path) const {
+  int value = 0;
+
+  if (!connected()) {
+    return value;
+  }
+
+  bat_ads_client_->GetIntegerPref(path, &value);
+  return value;
+}
+
+void BatAdsClientMojoBridge::SetIntegerPref(
+    const std::string& path,
+    const int value) {
+  if (!connected()) {
+    return;
+  }
+
+  bat_ads_client_->SetIntegerPref(path, value);
+}
+
+double BatAdsClientMojoBridge::GetDoublePref(
+    const std::string& path) const {
+  double value = 0.0;
+
+  if (!connected()) {
+    return value;
+  }
+
+  bat_ads_client_->GetDoublePref(path, &value);
+  return value;
+}
+
+void BatAdsClientMojoBridge::SetDoublePref(
+    const std::string& path,
+    const double value) {
+  if (!connected()) {
+    return;
+  }
+
+  bat_ads_client_->SetDoublePref(path, value);
+}
+
+std::string BatAdsClientMojoBridge::GetStringPref(
+    const std::string& path) const {
+  std::string value;
+
+  if (!connected()) {
+    return value;
+  }
+
+  bat_ads_client_->GetStringPref(path, &value);
+  return value;
+}
+
+void BatAdsClientMojoBridge::SetStringPref(
+    const std::string& path,
+    const std::string& value) {
+  if (!connected()) {
+    return;
+  }
+
+  bat_ads_client_->SetStringPref(path, value);
+}
+
+int64_t BatAdsClientMojoBridge::GetInt64Pref(
+    const std::string& path) const {
+  int64_t value = 0;
+
+  if (!connected()) {
+    return value;
+  }
+
+  bat_ads_client_->GetInt64Pref(path, &value);
+  return value;
+}
+
+void BatAdsClientMojoBridge::SetInt64Pref(
+    const std::string& path,
+    const int64_t value) {
+  if (!connected()) {
+    return;
+  }
+
+  bat_ads_client_->SetInt64Pref(path, value);
+}
+
+uint64_t BatAdsClientMojoBridge::GetUint64Pref(
+    const std::string& path) const {
+  uint64_t value = 0;
+
+  if (!connected()) {
+    return value;
+  }
+
+  bat_ads_client_->GetUint64Pref(path, &value);
+  return value;
+}
+
+void BatAdsClientMojoBridge::SetUint64Pref(
+    const std::string& path,
+    const uint64_t value) {
+  if (!connected()) {
+    return;
```

Comment:
are you sure we just want to ignore this? Seems like maybe we should return true/false?

Reply:
We currently follow the same API calls as `PrefService` however I will raise a separate issue as makes sense